README.md

```diff
@@ -1,31 +1,22 @@
-# LabFlow
+# Flow by LabGrid
 
-An open-source configurable LIMS for biotech labs.
+Flow by LabGrid is an open-source configurable workflow manager with a React/Typescript frontend, Python Flask backend, and Postgres database.  
 
-## Development
+This project is currently under active development. See the getting started documentation to run the app using Docker and Auth0 for user auth. Hosted version coming soon.
 
-Development docker containers that support hot code reloading on edit are provided.
+## Philosophy
 
-To run labflow in development mode:
+We want to make a self-hosted workflow manager for companies and biotech labs that have privacy and security concerns about their data going to third party services. We are striving for an easy-to-deploy solution that can be used in regulated lab environments. Flow by LabGrid gives you full control over your lab's workflows and data. 
 
-```
-# First, setup your environment.
-cp example.env .env
+## Features
+- **Custom workflow manager** - a customizable workflow manager you can deploy in your own environment 
+- **Drag and drop protocol widgets** - easy to make new workflow widgets
+- **Audit trails and reporting** - reports available for all events
+- **React Native support** - (work in progress)
+- **For your whole team** - share protocols with teammates, view runs across the team
+- **Built with Postgres, React, Flask** - optimized for responsiveness, fault-tolerance, and support for regulated environments. Backend based on Python means easy integration of scientific libraries.
 
-# Edit the .env file to contain secrets appropriate for your environment.
+## Documentation
 
-docker-compose build
-docker-compose up
-```
+[Getting started with development](https://github.com/lab-grid/flow/wiki/Getting-Started)
 
-Once the database, server, and webapp are running, the database itself needs to have its schema setup/updated.
```

Comment:
I usually like to leave at least the instructions to get the project up and running locally on the README.md since that's pretty much always the first thing anyone needs to do. In-depth descriptions can be kept in the wiki

Reply:
Ok I'll move the instructions back. I was following examples like [this](https://github.com/papercups-io/papercups). But looking at that again, they do indeed have a different "first thing anyone needs to do" (one-click deploy in their case). So we should keep the instructions in while that's the main thing to do...